go/connector/run.go

```diff
@@ -209,22 +288,25 @@ func runCommand(
 		}
 	}(cmd.Process.Signal)
 
-	err = cmd.Wait()
-	var closeErr = cmd.Stdout.(io.Closer).Close()
-	// Ignore error on closing stderr because it's already logged by the forwarder
+	var waitErr = cmd.Wait()
+	if waitErr != nil {
```

Comment:
nit: Maybe `if waitErr := cmd.Wait(); waitErr != nil {` since this error value isn't used past the body of this if statement?